tools/config/__init__.py

```diff
@@ -69,7 +69,7 @@
 
 
 ALLOWED_FEATURES = [
-    "BOOTLOADER", "BLE", "LWIP", "STORAGE", "NANOSTACK", "CRYPTOCELL310",
+    "BOOTLOADER", "BLE", "LWIP", "STORAGE", "NANOSTACK", "CRYPTOCELL310", "SSS",
```

Comment:
Should be removed (see my comment above about board specific features and subsystems).

Reply:
Addressed in the latest fc42210